package.json

```diff
@@ -1451,7 +1484,12 @@
         },
         {
           "command": "cmake.projectStatus.selectConfigurePreset",
-          "when": "view == cmake.projectStatus && cmake:enableFullFeatureSet && viewItem == 'configPreset'",
+          "when": "view == cmake.projectStatus && cmake:enableFullFeatureSet && viewItem =~ /configPreset/",
```

Comment:
do you need quotes around the `/configPreset/`

Reply:
No. The `=~` operator asks for regex, so the right hand side needs to be regex, you can see the documentation that I followed here: https://code.visualstudio.com/api/references/when-clause-contexts#match-operator